VideoLocker/res/drawable-v21/light_selectable_rounded_box_overlay.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Yes, we should use the ripple for touch feedback on anything tappable, unless we hear otherwise. It is the de-facto standard way of handling touch feedback on Android nowadays.